src/App.tsx

```diff
@@ -1,9 +1,25 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies = moviesFromServer.filter(movie => {
+    const { title, description } = movie;
+    const preparedTitle = title.toLowerCase();
+    const preparedDescription = description.toLowerCase();
+    const preparedQuery = query.trim().toLowerCase();
```

Comment:
Let's move this variable out of the filter method.😃
We need it to avoid creating a new variable at each iteration.